docs/classes/Refold.html

```
@@ -76,15 +76,15 @@ <h5 id="token"><code class="tsd-tag">Optional</code><span class="tsd-kind-proper
 </div></li></ul></li></ul></div>
 <h4 class="tsd-returns-title">Returns <a href="" class="tsd-signature-type tsd-kind-class">Refold</a></h4><aside class="tsd-sources">
 <ul>
-<li>Defined in <a href="https://github.com/gocobalt/refold-js/blob/55aa0963c1c116dae1307d9aeb7adca08c2d200d/refold.ts#L428">refold.ts:428</a></li></ul></aside></div></li></ul></section></section></details>
+<li>Defined in <a href="https://github.com/gocobalt/refold-js/blob/0845fcc4eb2152b8c3817b97d9540cd86946bff4/refold.ts#L448">refold.ts:448</a></li></ul></aside></div></li></ul></section></section></details>
 <details class="tsd-panel-group tsd-member-group tsd-accordion" open><summary class="tsd-accordion-summary" data-key="section-Properties"><svg width="20" height="20" viewBox="0 0 24 24" fill="none" aria-hidden="true"><use href="../assets/icons.svg#icon-chevronDown"></use></svg>
 <h2 class="tsd-anchor-link" id="properties-1">Properties<a href="#properties-1" aria-label="Permalink" class="tsd-anchor-icon"><svg viewBox="0 0 24 24" aria-hidden="true"><use href="../assets/icons.svg#icon-anchor"></use></svg></a></h2></summary>
 <section>
 <section class="tsd-panel tsd-member">
 <h3 class="tsd-anchor-link" id="token"><span>token</span><a href="#token" aria-label="Permalink" class="tsd-anchor-icon"><svg viewBox="0 0 24 24" aria-hidden="true"><use href="../assets/icons.svg#icon-anchor"></use></svg></a></h3>
```

HTMLHint (1.9.2)

[error] The id value [ token ] must be unique.

(id-unique)